bosh/manifest.yml

```diff
@@ -30,9 +30,9 @@ instance_groups:
   - name: consul_agent
     release: consul
     consumes:
-      consul_common: {from: consul_common_link, deployment: (( param "specify cf deployment" ))}
+      consul_common: ((cf-deployment))
```

Comment:
Looks like we're changing this value from

```
{from: consul_common_link, deployment: ((cf_deployment))}
```

to 

```
((cf_deployment))
```

Is that intentional?

Reply:
Not sure what you mean there?  The before was `{from: consul_common_link, deployment: (( param "specify cf deployment" ))}`, which is a spruce thing, which we are trying to get rid of, so I replaced it with a varsfile variable.  Is there a better way to do that?

Reply:
The `(( param "specify cf deployment" ))` piece uses spruce, but the rest of the line is plain yaml.

Reply:
Huh.  Never seen that before.  Is there some documentation on what that from: stuff is, then?

Reply:
https://bosh.io/docs/links/